src/api/manager/resolvers.js

```diff
@@ -561,9 +575,167 @@ const resolvers = {
         console.log(e)
       }
     },
+    addMultiRecords: async (_, { name, records }, { cache }) => {
+      const ens = getENS()
+
+      function setupTransactions({ name, records, resolverInstance }) {
+        try {
+          const resolver = resolverInstance.interface.functions
+          const namehash = getNamehash(name)
+          console.log('**** records', records)
+          const transactionArray = records.map((record, i) => {
+            switch (i) {
+              case 0:
+                if (!record) return undefined
```

Comment:
Probably this can move up to the first line of the function as everything in this function seem to depend on the existence of `record`